addons/mail/views/mail_activity_views.xml

```diff
@@ -332,4 +344,85 @@
         </field>
     </record>
 
+    <record id="mail_activity_open_target_view_tree" model="ir.ui.view">
```

Comment:
To define after 'mail_activity_view_tree_without_record_access' so that we group views by type, easier to find the one we need